shuttering/prod/end-civil-partnership-service-gov-uk.yml

```diff
@@ -9,7 +9,7 @@ shutter_all: false
 # - alias_target_resource_id
 shutter_all_a: false
 
-shutter_all_cname: false
+shutter_all_cname: true
```

Comment:
As this one includes a "*" records we need to set this to false. Can enable shuttering by setting the individual "www" record to true

Reply:
done